src/dapps/selectors.ts

```diff
@@ -23,6 +21,8 @@ export const dappsCategoriesSelector = (state: RootState) => state.dapps.dappsCa
 
 export const dappsListSelector = (state: RootState) => state.dapps.dappsList
 
+export const dappsV2ListSelector = (state: RootState) => state.dapps.dappsList.filter(isDappV2)
```

Comment:
Is it possible for dappsList to include both v1 and v2 dapps? Or is it always one or the other?
Also, should the other selector return v1? Or it's usages can work with both v1 and v2 schema?

Reply:
`dappListSelector` returns the dapps list regardless of if it's v1 or v2, that's okay for most cases where we just care about the dapp and not the category (e.g. featured, favourited, recently used etc)

it's not possible for the dappsList to include both v1 and v2 dapps, i actually took a shortcut here in the selector. what i really want is for the dappsList to be cleared when the experiment is switched on for the first time since the new screens is going to expect a different data format for rendering. i could either watch the experiment and clear the store or just not return the dappsList until the fetch is complete / the store is refreshed with new data. i opted for the latter because it is easier and also i don't know how to watch statsig experiment variables lol 😅